lib/slayer/errors.rb

```diff
@@ -18,6 +18,8 @@ def initialize(message = nil)
       super message
     end
   end
+	
+  class FormValidationException < StandardError; end
```

Comment:
Can we use "Error" instead of "Exception" to normalize with my Service error and StandardError?